src/shogun/classifier/svm/NewtonSVM.cpp

```diff
@@ -13,33 +13,35 @@
 #include <shogun/labels/BinaryLabels.h>
 #include <shogun/labels/Labels.h>
 #include <shogun/lib/Signal.h>
-#include <shogun/machine/LinearMachine.h>
 #include <shogun/mathematics/Math.h>
-#include <shogun/mathematics/lapack.h>
 #include <shogun/mathematics/linalg/LinalgNamespace.h>
 
 //#define DEBUG_NEWTON
 //#define V_NEWTON
 using namespace shogun;
 
-CNewtonSVM::CNewtonSVM() : CLinearMachine(true)
+CNewtonSVM::CNewtonSVM() : CIterativeMachine<CLinearMachine>()
```

Comment:
one thing: as `CIterativeMachine` is not available from swig, you wont be able to instantiate `NewtonSVM` from Python, so you can actually remove newton SVM from the interface files, see #4354